Sources/SwiftCrossUI/ViewGraph/PreferenceValues.swift

```diff
@@ -21,5 +52,13 @@ public struct PreferenceValues: Sendable {
                 }
             }
         }
+
+        // For presentation modifiers, take the first (top-level) value only
```

Comment:
`// For presentation modifiers, take the outer-most value (using child ordering to break ties).`